DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -240,13 +240,9 @@ function InstallPackage
         $chocoinstallparams += " $cParams"
     }
     Write-Verbose -Message "Install command: 'choco install $pName $chocoinstallparams'"
-
-    $packageInstallOuput = Invoke-Expression -Command "choco install $pName $chocoinstallparams"
+    $packageInstallOuput = Invoke-ChocoLatey "install $pName $chocoinstallparams"

```

Comment:
Picky but can you change Invoke-ChocoLatey to a lowercase 'l'?
Can you also add the arguments parameter explicitly?